apps/mobile/src/store/keychains/bitcoin/bitcoin-keychains.read.ts

```diff
@@ -132,3 +132,10 @@ export function useBitcoinAddresses() {
     [accounts]
   );
 }
+
+export function deriveBitcoinAccountFromAddress(
```

Comment:
```suggestion
export function deriveBitcoinAddressFromAccount(
```

Reply:
also can we say index zero address here, because the name infers 1:1 relationship between account and address.